File: javascript/src/models/eventTypeImportOpenApiOut.ts
// this file is @generated
import {
  type EventTypeImportOpenApiOutData,
  EventTypeImportOpenApiOutDataSerializer,
} from "./eventTypeImportOpenApiOutData";

export interface EventTypeImportOpenApiOut {
  data: EventTypeImportOpenApiOutData;
}

export const EventTypeImportOpenApiOutSerializer = {
  _fromJsonObject(object: any): EventTypeImportOpenApiOut {
    return {
      data: EventTypeImportOpenApiOutDataSerializer._fromJsonObject(object["data"]),
    };
  },

  _toJsonObject(self: EventTypeImportOpenApiOut): any {
    return {
      data: EventTypeImportOpenApiOutDataSerializer._toJsonObject(self.data),
    };
  },
};

File: rust/src/models/event_type_import_open_api_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

use super::event_type_import_open_api_out_data::EventTypeImportOpenApiOutData;

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct EventTypeImportOpenApiOut {
    pub data: EventTypeImportOpenApiOutData,
}
